tests/test-sinkinterface.c

```diff
@@ -133,24 +133,25 @@ iface_prepare (NSinkInterface *iface, NRequest *request)
 
 START_TEST (test_resynchronize)
 {
-    NSinkInterface *iface = NULL;
-    iface = g_new0 (NSinkInterface, 1);
-    ck_assert (iface != NULL);
-    const char *name = "TEST_RESYNC_sink_name";
-    iface->name = name;
-    NRequest *request = NULL;
-    request = n_request_new ();
-    ck_assert (request != NULL);
-    const char *req_name = "TEST_RESYNC_REQUST_name";
-    request->name = g_strdup (req_name);
-    request->sinks_preparing = NULL;
-    NCore *core = NULL;
-    core = n_core_new (NULL, NULL);
+    const char *fake_req_name = "TEST_RESYNC_requst_name";
```

Comment:
Typo in the name requst -> request

Reply:
yup, from original code. should be all gone now.